ocaml/typing/solver_intf.mli

```diff
@@ -178,7 +178,7 @@ module type Lattices_mono = sig
     ('a0, 'a1) Misc.eq option
 
   (** Print morphism *)
-  val print_morph : 'b obj -> Format.formatter -> ('a, 'b, 'd) morph -> unit
+  val print_morph : Format.formatter -> ('a, 'b, 'd) morph -> unit
```

Comment:
`morph` alone is not sufficient to uniquely identifies a morphism - it needs to be accompanied by the destination target.

For example, current `Const_min` is printed as just `const_min` - but I think it would be much more informative if we print `const_min_locality` instead.

Even if our `mode.ml` implementation doesn't need `dst`, we should keep the original interface of `solver_intf.mli` which is more correct.

Reply:
Nope -- not after perf gain. After simplicity. But I think you're right about keeping the interface with the obj. I'll make that change.